src/app/components/edit-resource/edit-resource.component.html

```diff
@@ -136,6 +136,17 @@
       &nbsp;
       <label for="hidden">Hidden</label>
     </div>
+
+    <div class="col">
+      <input
+        id="spotlight"
+        placeholder="spotlight"
+        type="checkbox"
+        [(ngModel)]="resource['attr-spotlight']"
+      />
+      &nbsp;
+      <label for="spotlight">Spotlight</label>
+    </div>
```

Comment:
move it before `Hidden` and use `col-3` for the column class (on both spotlight and hidden)